installer_scripts/install-yb-voyager

```diff
@@ -314,11 +315,26 @@ update_yb_voyager_bashrc() {
 
 
 install_yb_voyager() {
+	os=$1
 	output "Installing yb-voyager:${VERSION}."
 	if [ "${VERSION}" == "latest" ]
 	then
-		$GO install github.com/yugabyte/yb-voyager/yb-voyager@${YB_VOYAGER_GIT_HASH}
```

Comment:
After this do we still need `install_golang`?

Reply:
We should not I guess. We will be just running the binary.